BlueElectrum.js

```diff
@@ -354,10 +355,26 @@ module.exports.getDustValue = async () => {
 module.exports.broadcast = async function(hex) {
   if (!mainClient) throw new Error('Electrum client is not connected');
   try {
-    const broadcast = await mainClient.blockchainTransaction_broadcast(hex);
-    return broadcast;
+    return await mainClient.blockchainTransaction_broadcast(hex);
   } catch (error) {
-    return error;
+    if (error?.code === 1) {
+      const message = error.message.split('\n');
+      const generalMsg = message[0];
+      const detailedMsg = message[2];
+      const errorMsg = `${generalMsg}: ${detailedMsg}`;
```

Comment:
I'm wondering if it's really necessary. Wouldn't be enough `error.message.includes(messages.txnMempoolConflictCode18)`? I mean, we don't need to split it here and format the message. It simply doesn't seem like the responsibility of the function to make operations on strings.

Also I'd throw `GeneralServerError(error.message)` instead of `Error(errorMsg)` and I'd move the logic of formatting the message into `GeneralServerError`.

Reply:
added BroadcastError Server is too general